src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -169,7 +169,7 @@ var HTMLDOMPropertyConfig = {
     // Setting .type throws on non-<input> tags
     type: MUST_USE_ATTRIBUTE,
     useMap: null,
-    value: MUST_USE_PROPERTY | HAS_SIDE_EFFECTS,
+    value: MUST_USE_ATTRIBUTE,
```

Comment:
Already mentioned here: https://github.com/facebook/react/pull/5680#issuecomment-165274589, resolution was that HAS_SIDE_EFFECTS can be handled as a followup PR.